lib/index.d.ts

```diff
@@ -1415,6 +1415,14 @@ declare namespace Stardog {
          * @param {string} accessKey the accessKey that was assigned to the credential when it was added
          */
         function removeCredential(conn: Connection, accessKey: string): Promise<HTTP.Body>;
+
+        /**
+         * Tests a provider connection
```

Comment:
```suggestion
         * Tests a provider connection using existing/new credentials.
```

`npm run docs` would be overwritten if not changed here as well